nengo_dl/tests/test_setup.py

```diff
@@ -8,18 +8,18 @@
 
 import nengo_dl
 
-repo_root = pathlib.Path(nengo_dl.__file__).parent.parent
-setup_py_path = repo_root / "setup.py"
-
 
 @pytest.mark.skipif(
     sys.version_info < (3, 8, 0),
     reason="ast.parse `feature_version` added in Python 3.8",
 )
-def test_setup_python35_compat():
+@pytest.mark.parametrize("feature_version", [(3, 0), (3, 1), (3, 2), (3, 4), (3, 5)])
```

Comment:
According to [the documentation](https://docs.python.org/3/library/ast.html#ast.parse), the lowest supported version for `ast.parse` is 3.4, so maybe we shouldn't go below that.

Reply:
Though it does seem to run without error for those lower versions, so :shrug:

Reply:
Yeah I suspect setup.py wouldn't actually run correctly (even if it doesn't contain actual syntax errors) on those old versions anyway, so probably not much point checking them.